user-service/user-infra/src/main/kotlin/com/grepp/quizy/user/infra/user/repository/UserRepositoryAdaptor.kt

```diff
@@ -1,20 +1,21 @@
-package com.grepp.quizy.infra.user.repository
+package com.grepp.quizy.user.infra.user.repository
 
-import com.grepp.quizy.domain.user.User
-import com.grepp.quizy.domain.user.UserRepository
-import com.grepp.quizy.infra.user.entity.UserEntity
-import com.grepp.quizy.infra.user.exception.UserNotFoundException
-import jakarta.transaction.Transactional
+import com.grepp.quizy.user.domain.user.User
+import com.grepp.quizy.user.domain.user.UserRepository
+import com.grepp.quizy.user.infra.user.entity.UserEntity
 import org.springframework.stereotype.Repository
 import kotlin.jvm.optionals.getOrNull
 
 @Repository
-class UserRepositoryAdaptor (
+class UserRepositoryAdaptor(
     private val userJPARepository: UserJPARepository
 ) : UserRepository {
 
-    override fun findById(id: Long): User =
-        userJPARepository.findById(id).getOrNull()?.toDomain() ?: throw UserNotFoundException()
+    override fun findById(id: Long): User? =
+        userJPARepository.findById(id).getOrNull()?.toDomain()
```

Comment:
애초에 이런게 있더라구요
```suggestion
    override fun findById(id: Long): User? =
        userJPARepository.findByIdOrNull(id)?.toDomain()
```

한번 써보시는거 추천해드려요

Reply:
아하 JPA에서 없으면 null로 반환해주는게 있나보네요 적용시켜보겠습니다!